scripts/create_output_index_json.py

```diff
@@ -16,24 +16,34 @@ def get_global_subitems():
         items = json.load(json_fh)
     return items
 
-def build_tab_from_json(tab, adoc_dir):
+def build_tab_from_json(tab, adoc_dir, img_dir):
     json_path = os.path.join(adoc_dir, tab['from_json'])
+    tab_key = tab['directory']
+    box_images_dir = os.path.join(img_dir, "full-sized")
+    available_images = os.listdir(box_images_dir)
+    available_images = sorted([f for f in available_images if re.search(tab_key+"_", f) is not None])
     with open(json_path) as json_fh:
         tab_data = json.load(json_fh)
         tab['subitems'] = []
+        counter = 0
         for item in tab_data:
             newsubitem = {}
             newsubitem['title'] = tab_data[item]['name']
             newsubitem['description'] = tab_data[item]['description']
             newsubitem['subpath'] = item + ".adoc"
-            newsubitem['imagepath'] = os.path.join('/images', 'placeholder/placeholder_square.png')
+            if len(available_images) > counter:
+                newsubitem['imagepath'] = os.path.join('/images', 'full-sized', available_images[counter])
+            else:
+                newsubitem['imagepath'] = os.path.join('/images', 'placeholder/placeholder_square.png')
```

Comment:
@nelliemckesson I _think_ this is saying that if we have 6 available_images, but 8 items on the page, then items 7 and 8 will be given the placeholder image? IMHO it'd be better to cycle round and give image1 to item7 and image2 to item8? i.e. something like:
```python
newsubitem['imagepath'] = os.path.join('/images', 'full-sized', available_images[ counter % num_available_images ])
```
:question:

Reply:
Ah, but in that case we probably want the "spanner" image to be treated separately to the "hex" images? :thinking:

Reply:
The spanner image is for the "getting started" box. The rest of the images are for the other boxes (and it doesn't really matter which one they get)

Reply:
@lurch Since @nelliemckesson won't be up for a few more hours, can you go in a tweak this please so you're happy with it. Lets get this merged down.